conflicts/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 
 [dependencies]
 clap = { version = "4", features = ["derive"] }
-octocrab = { git = "https://github.com/XAMPPRocky/octocrab", branch = "master" }
+octocrab = { git = "https://github.com/XAMPPRocky/octocrab", rev = "181dc166ae3607b6299b0de2350456428b467598" }
```

Comment:
You can run `cargo update` instead of modifying this

Reply:
Thanks, ran cargo update in every crate instead.

Reply:
No need to run it in every crate, only the one that needs it ;)

Reply:
Last time I did this, the util crate wouldn't compile with webhook_features because of conflicting crates so I upgraded the util crate. And then other crates wouldn't compile because of the util crate :grimacing:.

Reply:
Yeah, but I think then you modified the `toml` file, not the `lock` file. As long as there are no conflicts in the `toml` files (including the ones of all dependencies), it should be fine to have different `lock` files.

Reply:
Oh I see, will try this thanks.

Reply:
Yeah, that's better :+1: